### ISSUE TITLE
Filtering on multiple services does not work

### ISSUE THREAD

**Issue by brettjacobson** [andrewmostello/prometheus-windows-exporter.install#8]
Desired outcome:  collect only a few services, not all windows services

According to windows_exporter docs, to filter and provide metrics for only a few select services, an `OR` statement is required.

From their [Flags docs](https://github.com/prometheus-community/windows_exporter/blob/5931604b5851ea659526303fb5d3a6ce44170538/docs/collector.service.md#flags):

```
--collector.service.services-where
A WMI filter on which services to include. Recommended to keep down number of returned metrics.

Example: --collector.service.services-where="Name='windows_exporter'"

Example config win_exporter.yml for multiple services: services-where: Name='SQLServer' OR Name='Couchbase' OR Name='Spooler' OR Name='ActiveMQ'
```

The problem seems to be the way Chocolatey parses the `--package-parameters` options, with all the `=` and ` ` and ` OR `'s.  I cannot figure out a suitable quoting mechanism that does not get stripped out and keeps the proper stuff together to pass onto msiexec `EXTRA_FLAGS`.

This package parameter works: `$extraFlags = "/ExtraFlags:--collector.service.services-where=Name=docker"` (because there are no spaces and no special quoting attempted).

However, I've been unable to get any variation of this to succeed: `$extraFlags = "/ExtraFlags:--collector.service.services-where='Name=docker OR Name=vmcompute'"`.  This results in the msiexec popup error window showing install options because there is an extra ` OR Name=vmcompute` on the CLI passed to msiexec; the choco output shows `Extra flags: '--collector.service.services-where='Name=docker OR Name=vmcompute'` but you'll notice there are 3 apostrophes on that output, the begin and end are explicitly written by the Write-Host statement at https://github.com/andrewmostello/prometheus-windows-exporter.install/blob/master/tools/chocolateyinstall.ps1#L39.  My final apostrophe after vmcompute was eaten by Chocolatey parameter parsing I believe.

It seems like no amount of quoting prevents Chocolatey parameter parsing to respect a space anywhere in the /ExtraFlags option.

I've tried a million combinations of quotes, apostrophes and all, with no luck to get an OR to work on this statement, I'm looking for some help to properly make use of this feature.

**Comment by brettjacobson** [comment]
We will likely move to using a config file, but the config filename is also specified via ExtraFlags, and if the file path has a space in it, same problem :(

**Comment by andrewmostello** [comment]
@brettjacobson You're right about all this, and I can't get it to work either. I tried a few things to escape the inputs in the install script, as well as the input command, but it's still not happy. I think in the long run here we need to move way from using the msi installer.
